src/services/ComputerMind/ComputerMind.js

```diff
@@ -0,0 +1,48 @@
+
+export class ComputerMind {
+    //ustawianie prawdopodobienstwa
+    setGameLevel(randomObj) {
```

Comment:
to przyjmuje nextQuestion w wywołaniu, co więcej mówi niż randomObj, starajmy się nazywać parametry tak, żeby było wiadomo co przychodzi. wtedy od razu wiesz gdzie tego szukać/jakie ma pola itd

Reply:
poprawione:)